src/ticket-history/TicketHistoryScreenComponent.tsx

```diff
@@ -8,17 +8,13 @@ import {
   useTicketingState,
 } from '@atb/ticketing';
 import {useTimeContextState} from '@atb/time';
-import {
-  TicketingTexts,
-  TranslateFunction,
-  useTranslation,
-} from '@atb/translations';
+import {TicketingTexts, useTranslation} from '@atb/translations';
 import {View} from 'react-native';
 import {RefreshControl} from 'react-native-gesture-handler';
+import {TicketHistoryMode} from '@atb/ticket-history';
+import {TicketHistoryModeTexts} from '@atb/translations/screens/Ticketing';
 
-type Mode = 'expired' | 'sent';
-
-type Props = {mode: Mode};
+type Props = {mode: TicketHistoryMode};
```

Comment:
Could we just remove this `Props` type and use `TicketHistoryScreenParams` instead?

Reply:
done!